gapic-generator-java/src/main/java/com/google/api/generator/Main.java

```diff
@@ -26,6 +27,8 @@ public static void main(String[] args) throws IOException {
     ProtoRegistry.registerAllExtensions(registry);
     CodeGeneratorRequest request = CodeGeneratorRequest.parseFrom(System.in, registry);
     CodeGeneratorResponse response = Generator.generateGapic(request);
+    CodeGeneratorResponse springResponse = SpringGenerator.generateSpring(request);
```

Comment:
We shouldn't need these changes as SpringCodeGen are generated through the `Main` class in `spring` folder

Reply:
Ah yes thanks for catching this - removed changes to this file.